src/Sulu/Component/SmartContent/Configuration/BuilderInterface.php

```diff
@@ -27,6 +27,13 @@ public static function create();
      */
     public function enableTags(bool $enable = true);
 
+    /**
+     * @param mixed[] $types
+     *
+     * @return BuilderInterface
+     */
+    public function enableTypes(array $types = []);
```

Comment:
Strictly speaking that would be a BC break... Not sure if it matters too much here, I don't think so, because the `Builder` itself is just used by other `SmartContentDataProviders` and this interface is probably not implemented outside of Sulu.